sysutils/portfile-gen/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        macports macports-contrib pypi2port-20230621
+
+name                portfile-gen
+version             0.2
+revision            0
+categories          sysutils macports
+maintainers         {gwmail.gwu.edu:egall @cooljeanius} openmaintainer
+platforms           darwin
```

Comment:
Should be `any`.

Reply:
Fixed in [cba79b7](https://github.com/macports/macports-ports/pull/20951/commits/cba79b7f4ec177d2b63630c0a4e76747d41106ca).